### PR TITLE
spread-shellcheck: silly fix & pep8

### PR DESCRIPTION
**Pull request by bboozzoo**
Fix a bug when a variable is referenced before assignment. `pylint` had not caught that, boo.

Apply some pep8 formatting.

### CODE REVIEW

File: spread-shellcheck [hunk 1]
```diff
@@ -188,7 +193,6 @@ def main(opts):
                 logging.error('validation failed for required files')
                 raise SystemExit(1)
 
-        logging.debug('must validate: %s', musts)
```
Comment (pedronis) [inline spread-shellcheck]: shouldn't this still be logged somewhere? or is it already?

**Comment by pedronis** [comment]
unrelated to this change but I was looking at the code, 

ShellcheckFailures.contains  should probably be called something like  contains_any

File: spread-shellcheck [hunk 2]
```diff
@@ -83,7 +86,10 @@ class ShellcheckFailures(Exception):
         return len(self.failures)
 
     def contains(self, other):
```
Comment (pedronis) [inline spread-shellcheck]: this name was and is a bit confusing I think

Comment (bboozzoo) [reply spread-shellcheck]: Now that the method has changed a little bit maybe `contained` or just `intersection` would work?

Comment (pedronis) [reply spread-shellcheck]: probably obvious is best, 'intersection'

**Pull request merged**